src/test/app/RCLValidations_test.cpp

```diff
@@ -108,7 +108,8 @@ class RCLValidations_test : public beast::unit_test::suite
             next->updateSkipList();
             if (forceHash)
             {
-                next->setImmutable(config);
+                BEAST_EXPECT(next->read(keylet::fees()));
```

Comment:
Why is this expected only when `forceHash` is set? I didn't go into the details of this test but the `fees` is available regardless of `forceHash`.